tests/strategies/external/utils.py

```diff
@@ -15,17 +15,19 @@
 You should have received a copy of the GNU General Public License
 along with this program.  If not, see <http://www.gnu.org/licenses/>.
 """
+# pylint: disable=missing-function-docstring
```

Comment:
Nit: this module is not a test module, but rather a utils module (for tests). So I wouldn't disable this warning.